chapter2/tests/t20/terraform/base/acm.tf

```diff
@@ -0,0 +1,17 @@
+resource "aws_acm_certificate" "certificate" {
+  domain_name       = var.site_url
+  validation_method = "DNS"
+  subject_alternative_names = [
+    "www.${var.site_url}"
+  ]
+
+  lifecycle {
+    create_before_destroy = true
+  }
+  tags = var.tags
```

Comment:
Please pass tags to all AWS resources from `aws` provider directly

Reply:
> Please pass tags to all AWS resources from `aws` provider directly

@kseniyashaydurova 
Done